src/software/util/enum_string_macro/BUILD

```diff
@@ -0,0 +1,6 @@
+package(default_visibility = ["//visibility:public"])
+
+cc_library(
+    name = "enum_string_macro",
+    hdrs = ["enum_string_macro.h"],
```

Comment:
I think `printable_enum_macro` (or something similar) might be a bit more descriptive?

Reply:
changed